### PR TITLE
Bump versions to support angular 6.0.0 and rxjs 6

### PR DESCRIPTION
**Pull request by ivalexa**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -95,13 +95,13 @@
...
-    "@angular/platform-browser-dynamic": "^5.0.0",
+    "@angular/common": "^6.0.0",
+    "@angular/compiler": "^6.0.0",
+    "@angular/core": "^6.0.0",
+    "@angular/platform-browser": "^6.0.0",
+    "@angular/platform-browser-dynamic": "^6.0.0",
     "core-js": "^2.4.1",
-    "rxjs": "^5.5.2",
+    "rxjs": "^6.1.0",
     "zone.js": "^0.8.14"
```
Comment (devoto13) [inline package.json]: Can you align zone.js [version](https://github.com/angular/angular/blob/master/packages/core/package.json#L20) with `@angular/core` as well, please?

**Comment by DaSchTour** [comment]
would be better to use >= 5.0.0 instead. Because the current change would mean that 5.0.0 isn't supported anymore

**Comment by scttcper** [comment]
@DaSchTour angular 5 continues to be supported even though it is compiled with angular 6

**Comment by scttcper** [comment]
they aren't included as actual dependencies in the final result. the current package.json has 

```json
{
  "name": "angular-fontawesome",
  "version": "1.0.0",
  "author": "Paul Marbach <paul@picardylearning.com>",
  "description": "Angular directive for FontAwesome",
  "scripts": {
    "test": "./node_modules/bower/bin/bower install; ./node_modules/karma/bin/karma start --browsers Firefox --single-run"
  },
  "main": "dist/angular-fontawesome.js",
  "repository": {
    "type": "git",
    "url": "https://github.com/picardy/angular-fontawesome.git"
  },
  "bugs": {
    "url": "https://github.com/picardy/angular-fontawesome/issues"
  },
  "keywords": [
    "angular",
    "fontawesome",
    "directive"
  ],
  "devDependencies": {
    "bower": "1.3.12",
    "grunt": "^0.4.4",
    "grunt-bump": "~0.0.15",
    "grunt-contrib-clean": "0.6.0",
    "grunt-contrib-connect": "^0.11.2",
    "grunt-contrib-copy": "0.8.0",
    "grunt-contrib-uglify": "0.8.0",
    "grunt-eslint": "^17.2.0",
    "grunt-gh-pages": "^0.10.0",
    "grunt-karma": "^0.8.2",
    "karma": "^0.12.9",
    "karma-chrome-launcher": "0.1.7",
    "karma-firefox-launcher": "0.1.4",
    "karma-jasmine": "0.3.5",
    "karma-ng-html2js-preprocessor": "^0.1.0",
    "karma-ng-scenario": "^0.1.0",
    "karma-phantomjs-launcher": "0.1.4",
    "karma-safari-launcher": "0.1.1"
  },
  "license": "MIT"
}
```

Probably have the required dep as peerDependencies though. Usually angular projects have a separate package.json that is used to ship with npm.

**Comment by DaSchTour** [comment]
@scttcper that's not the current package.json

**Comment by scttcper** [comment]
@DaSchTour i see. something is funky going on in the current one

```json
{
  ...
  "peerDependencies": {
    "@fortawesome/fontawesome-svg-core": "^1.2.0-9"
  },
  "peerDependencies": {
    "@angular/common": "^5.0.0",
    "@angular/compiler": "^5.0.0",
    "@angular/core": "^5.0.0",
    "@angular/platform-browser": "^5.0.0",
    "@angular/platform-browser-dynamic": "^5.0.0",
    "core-js": "^2.4.1",
    "rxjs": "^5.5.2",
    "zone.js": "^0.8.14"
  }
}
```

**Comment by crhistianramirez** [comment]
Is rxjs even needed as a dependency? I think we might be able to just get rid of it altogether.

I also think the dependency on Angular should read something like: 
```>=5.0.0 || >=6.0.0``` 

to support both

**Comment by robmadole** [comment]
We just released 0.1.0-10 that should have support for 5.0 and 6.0. Can you give this a try @ivalexa

**Comment by devoto13** [comment]
As @robmadole mentioned support for Angular 6 has been added. So I'm going to close this PR as obsolete.

@ivalexa Thanks for you contribution anyway!

**Pull request closed without merging**